doc/source/ray-air/experimental-features.rst

```diff
@@ -0,0 +1,136 @@
+.. _air-experimental-features:
+
+================================
+Experimental features in Ray AIR
+================================
+
+We're testing a number of experimental features in Ray AIR.
+
+During development and internal dogfooding, the features
+are disabled per default. You can opt-in by setting a
+feature-specific environment variable.
+
+After some time, we enable the feature per default to gather
+more feedback from the community. In that case, they can still
+be disabled using the same environment variable. This will
+fully revert to the old behavior, should the new behavior
+not suffice your needs.
+
+If you run into issues with experimental features,
+please `open an issue <https://github.com/ray-project/ray/issues/>`_
+on GitHub and let us know! This will give
+us the chance to incorporate your feedback before we remove
+the old implementation and make the new implementation the
+default.
+
+.. note::
+
+    Experimental features can undergo frequent changes,
+    especially on the master branch and the nightly wheels.
+
+.. _air-experimental-new-output:
+
+New output engine
```

Comment:
I would caution against using "new" in user documentation. It's a temporary condition and is obvious when the doc is has not been updated in a while.

Reply:
Renamed to "Context-aware"